tools/test/audio/test_utils/delete_check.m

```diff
@@ -30,6 +30,7 @@
 
 function delete_check(really, f)
 if really && exist(f) == 2
+% if really && exist(fullfile('',f),'file') == 2 This is sensitive, ATTENTION[really & f]
```

Comment:
There's . missing from fullfile() I think

Reply:
I'II push the code as per your suggestion but i am not sure of correctness of results.  When I debugged really and f output. The behavior was unexpected and  we see errors upon running the src_test(32,32)